### PR TITLE
Setup dependabot by including a dependabot.yml file

### PR DESCRIPTION
**Pull request by sarahraqueld**
There are a number of vulnerabilities with the current versions of our dependencies. Setting up dependabot will help us keep up to date with security patches and other dependency updates.  

Add a dependabot.yml configuration file. Up to 3 PRs will be opened by dependabot at once.

### CODE REVIEW

File: .github/workflows/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+version: 2
+updates:
+  - package-ecosystem: 'bundler'
+    directory: '/'
+    schedule:
+      interval: 'weekly'
+      time: '08:00'
+      timezone: America/New_York
+    open-pull-requests-limit: 3
```
Comment (littleforest) [inline .github/workflows/dependabot.yml:9]: Any reason not to have them just get created in real-time as they come up?

**Review by littleforest — COMMENTED** [review]
Thanks for setting this up!

File: .github/workflows/dependabot.yml (hunk 1 above)
Comment (sarahraqueld) [reply .github/workflows/dependabot.yml:9]: Good question, I don't have a good one, will remove the schedule (will keep the open prs limit so we don't get flooded with more PRs than we can merge

Comment (sarahraqueld) [reply .github/workflows/dependabot.yml:5]: done!

**Review by littleforest — APPROVED** [review]

**Pull request merged**